### ISSUE TITLE
Daily standup

### ISSUE THREAD

**Issue by jackgoh** [hackerspacemmu/hackerspace-bot#14]
**Is your feature request related to a problem? Please describe.**
Most of the time, when we work on project alone, we tend to procrastinate. Always promise ourself will do it tomorrow, and tomorrow to infinite. This daily standup bot helps to remind us to post an update for what we have done yesterday, what we going to do today and what roadblock we facing that others can see and perhaps help us to unblock. Also, to keep us active in our own project we said we going to build by sharing our todos every day for everyone to see and build peer pressure around

**Describe the solution you'd like**
A daily standup bot that we can update. Example: !dailystandup yesterday Login feature, Feature abc. Today, Post comment feature, etc

**Comment by bingyuanng** [comment]
👍🏻 I was thinking if it's a good idea for the bot to pm users for what they have done. And then send it into a separate channel. Or how does this work?

**Comment by jackgoh** [comment]
@bingyuanng yes, can discord bot achieve that ?

**Comment by bingyuanng** [comment]
Yes. Can private message users. How to initiate start? Not sure about the flow. Doesn't seem ok to remind everyone in the channel.
